Aptoide/aptoide-client-v8
# [bugfix/MOB-478] Notification Ready To Install (Android 10-11)

**What does this PR do?**

   This implements a new notification that appears when a download is finished in background to warn users that their app is ready. It uses the default notification system that we have with Aptoide but it uses a specific channel and it has a few quirks. One of those is the fact that we have to track if the notification is shown (manually) to change the type of notification we show. See the testing section to undestand the constraints.

**NOTE:** Strings are still hardcoded and must be fixed!

**Database changed?**

   No

**How should this be manually tested?**

  This PR implements two similar notifications that has to be tested. The constraints are as follows:
    - Both notifications will only appear when a download is finished and the app is in background (is either "minimized" or closed")
    - Both notifications will only appear for devices with Android 10 or newer (11 for now).

  There's two types of notifications:
    - Single app notification: This is a notification to warn the user that a specific app has finished downloading. It has the app name and icon. Clicking it will redirect to its appview and start the installation immediately.
    - Multi-app notification: This is a notification that only appears if another download is finished WHILE there's a single app notification (or multi) already active. If there is, the single app notification will transform to a multi-app notification. Clicking this will redirect to the Apps section. This is especially useful notification for "update all" in Apps.

**What are the relevant tickets?**

  [MOB-478](https://aptoide.atlassian.net/browse/MOB-478)

**Code Review Checklist**
